TaskBroker/VstsServerTaskBroker/Contracts/VstsMessage.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace VstsServerTaskHelper
```

Comment:
Should not we have 2 separate message classes, one for build and one for release?